### PR TITLE
W-10813219 Xero 1.1.0

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
W-10813219 Xero 1.1.0

### CODE REVIEW

File: xero/1.1/modules/ROOT/pages/xero-accounting-connector-reference.adoc [hunk 1]
```diff
@@ -0,0 +1,1257 @@
...
+* XERO-ACCOUNTING:TOO_MANY_REQUESTS 
+* XERO-ACCOUNTING:UNAUTHORIZED 
+* XERO-ACCOUNTING:UNSUPPORTED_MEDIA_TYPE 
+
+
+[[GetInvoices]]
+== Get Invoices
+`<xero-accounting:get-invoices>`
+
+Retrieves all invoices for a Xero organisation. This operation makes an HTTP GET request to the /Invoices endpoint.
```
Comment (sathya0) [inline xero/1.1/modules/ROOT/pages/xero-accounting-connector-reference.adoc]:
```suggestion
Retrieves all invoices for a Xero organization. This operation makes an HTTP GET request to the /Invoices endpoint.
```

File: xero/1.1/modules/ROOT/pages/xero-accounting-connector-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,1257 @@
...
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| where a| String |  Filter by an element. |  |
+| order a| String |  Order by an element. |  |
+| IDs a| Array of String |  Filter by a comma-separated list of contact IDs, which enables you to retrieve a specific set of contacts in a single call. |  |
+| page a| Number |  For example, `page=1`. Up to 100 contacts are returned in a single API call. |  1 |
+| Xero tenant ID a| String |  Xero identifier for the Tenant. |  | x
+| If Modified Since a| String |  Only records created or modified since this timestamp are returned. |  |
+| Output Mime Type a| String |  The mime type of the payload that this operation outputs. |  |
```
Comment (sathya0) [inline xero/1.1/modules/ROOT/pages/xero-accounting-connector-reference.adoc]:
```suggestion
| Output Mime Type a| String |  MIME type of the payload that this operation outputs. |  |
```

**Review by sathya0 — DISMISSED** [review]

File: xero/1.1/modules/ROOT/pages/xero-accounting-connector-reference.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply xero/1.1/modules/ROOT/pages/xero-accounting-connector-reference.adoc]: Fixed.

File: xero/1.1/modules/ROOT/pages/xero-accounting-connector-reference.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply xero/1.1/modules/ROOT/pages/xero-accounting-connector-reference.adoc]: Fixed.

**Review by sathya0 — APPROVED** [review]

**Pull request merged**